npm-shrinkwrap.json

```diff
@@ -1605,7 +1683,7 @@
         "isarray": {
           "version": "0.0.1",
           "from": "isarray@0.0.1",
-          "resolved": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz"
+          "resolved": "http://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz"
```

Comment:
Yes :(
I get the same result (for some reason, but @rcorral's setup gives him https for all. Not sure what is different with our environments)